### PR TITLE
gravel: add env for custom config path

### PR DESCRIPTION
**Pull request by mgfritch**
Adds environment var `AQUARIUM_CONFIG_DIR=` for specifying a custom
location to the aquarium config files

Resolves: #160 
Signed-off-by: Michael Fritch <mfritch@suse.com>

### CODE REVIEW

**Comment by mgfritch** [comment]
DNM: adding unit tests

**Review by jecluis — COMMENTED** [review]
lgtm, waiting for final review

**Review by jecluis — APPROVED** [review]

**Pull request merged**
